pom.xml

```diff
@@ -132,6 +132,13 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>${appengine.target.version}</version>
       </plugin>
+
+      <plugin>
+      	<groupId>com.google.appengine</groupId>
+	<artifactId>gcloud-maven-plugin</artifactId>
+      	<version>1.9.17</version>
```

Comment:
How about to update the value in $appengine.target.version in properties section and use ${appengine.target.version} here?